README.md

```diff
@@ -157,6 +157,13 @@ fn main() {
     window.show();
 
     event_loop.run_forever(|_event| {
+        match _event {
+            glutin::Event::WindowEvent { event, .. } => match event {
```

Comment:
Instead of `event`, you can directly write `glutin::WindowEvent::Closed`. Then you have one less `match` to use.